<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWindow</class>
 <widget class="QMainWindow" name="StartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1370</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Start Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="gridSizeLabel">
        <property name="text">
         <string>Select Grid Size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gridSizeComboBox">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>9x9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16x16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25x25</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="difficultyLabel">
        <property name="text">
         <string>Select Difficulty</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="difficultyComboBox">
        <item>
         <property name="text">
          <string>Easy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Extreme</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="numPlayersLabel">
        <property name="text">
         <string>Pick Player Names divided by ,   (up to 4 Players are possbile)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameInput"/>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="styleSheet">
         <string notr="true">color=rgb(239, 41, 41)</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start Game</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
